app/views/mentee_application_mailer/notify_for_code_challenge_approved.html.erb

```diff
@@ -0,0 +1,12 @@
+<div style='display: block; max-width: 36rem; margin: 0 auto; padding-top: 4rem;'>
+  <p><%= @recipient.full_name %>,</p>
+  <p>
+    We're happy to share that we're excited to move you forward in the process! 🎉
+  </p>
+  <p>
+    For the next step, we'd like to invite you to meet with Dave Paola, Founder of the Agency of Learning.
+  </p>
+  <p>
+    To book a time, please use this link: <a href="https://calendly.com/davepaola/agency-of-learning-interview">https://calendly.com/davepaola/agency-of-learning-interview</a>.
```

Comment:
waiting on @dpaola2 to finish defining this workflow/email.

Reply:
Can you use this link instead please: https://savvycal.com/davepaola/73e2db9f

Reply:
we might want to ENV this so people can't see it via us being open source (and we can easily change dynamically if needed).